modules/shared/ui/Layout/Header/Header.tsx

```diff
@@ -118,12 +118,18 @@ export function Header() {
         </InputWrap>
 
         <ActionsDesktop>
-          <Network>
-            <NetworkBulb color={getChainColor(chainId)} />
+          {chainId in CHAINS_COLORS ? (
```

Comment:
I think it's better to show bulb icon even if Unknown network is shown. So maybe we could tweak `getChainColor` and add a fallback to it, like this: 

```
const getChainColor = (chainId: CHAINS) => {
  try {
    return getChainColorSdk(chainId)
  } catch (error) {
    return getChainColorSdk(CHAINS.Kovan)
  }
}
```

We can even tweak `getChainName` and add a fallback to it:

```
export const getChainName = (chainId: number) =>
  get<string | undefined>(ChainNames, parseChainId(chainId)) ?? 'Unknown'
```

wdyt?